presto/tests/conftest.py

```diff
@@ -17,4 +18,9 @@ def dd_environment(instance):
 @pytest.fixture(scope='session', autouse=True)
 @pytest.mark.usefixtures('dd_environment')
 def instance():
-    return load_jmx_config()
+    inst = load_jmx_config()
+    # Add presto coordinator to the configuration
+    inst.get('instances').append(deepcopy(inst.get('instances')[0]))
+    inst['instances'][0]['port'] = 9997
```

Comment:
So we are monitoring the coordinator instead of the worker because it has more metrics? 
Does the worker has some non overlapping metrics, if yes, maybe it should be monitored too? (https://github.com/DataDog/integrations-core/blob/master/presto/tests/docker/docker-compose.yaml)

Should we replace the port here instead of overriding?
https://github.com/DataDog/integrations-core/blob/master/presto/datadog_checks/presto/data/conf.yaml.example#L23